packages/server/src/arpa_reporter/lib/audit-report.js

```diff
@@ -308,12 +308,13 @@ async function createReportsGroupedByProject(periodId, tenantId, dateFormat = RE
         projectLogger.debug('populating row from records in project');
 
         // set values for columns that are common across all records of projectId
-        const lastProjectRecord = projectRecords[projectRecords.length - 1];
+        const projectSheetRecords = projectRecords.filter((record) => Object.keys(EXPENDITURE_CATEGORIES).includes(record.type));
+        const lastProjectRecord = projectSheetRecords[projectSheetRecords.length - 1];
         const row = {
             'Project ID': projectId,
-            'Project Description': lastProjectRecord.content.Project_Description__c,
-            'Project Expenditure Category Group': ec(lastProjectRecord.type),
-            'Project Expenditure Category': lastProjectRecord.subcategory,
+            'Project Description': lastProjectRecord?.content.Project_Description__c,
+            'Project Expenditure Category Group': ec(lastProjectRecord?.type),
```

Comment:
What if this is null? Would that give an `undefined`? Do we want some sort of default value here?

Reply:
If `lastProjectRecord` is null, then these values become empty, yep.

I actually do like the idea of a default value... maybe something like `"project with ID not listed in any upload"`?